transcendence-app/src/user/user.controller.ts

```diff
@@ -33,6 +36,21 @@ export class UserController {
     return req.user;
   }
 
+  @Get('users')
+  getUsers() {
+    return this.userService.retrieveUsers();
+  }
+
+  @Get('usernames?')
+  getUsernames(
+    @Query('take', ParseIntPipe) take: number,
+    @Query('skip', ParseIntPipe) skip: number,
+    @Query('sort', ParseBoolPipe) sort: boolean,
```

Comment:
Could you make a brief explanation about what this params represent?

Reply:
Of course.
take is a number which represent the amount of register you want to fetch.
skip is also a number representing the starting point of the retrieve
sort is a boolean to indicate if you want the results order arranged.
For example:
[http://127.0.0.1:3000/api/v1/user/usernames?take=3&skip=1&sort=true](http://127.0.0.1:3000/api/v1/user/usernames?take=3&skip=1&sort=false) would return 3 users, skipping the first entry, and as they are in the array (database), without being sorted.